Tienda-Virtual/src/app/modules/product/component/category/category.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="body-content">
+    <div class=" card">
+        <div class="card-header">
+          <h1>Categorías
+          </h1>
+          <div class="card-body">
+            <table class="table table-dark">
+              <thead>
+                <tr>
+                  <th scope="col">Id</th>
+                  <th scope="col">Categoria</th>
+                  <th scope="col">Tag</th>
+                  <th scope="col">Estado</th>
+                </tr>
+              </thead>
+                <tbody>
+                        @for (item of categories; track item){
+                        <tr class="table-default text-white">
+                            <td>{{item.category_id}}</td>
+                            <td>{{item.category}}</td>
+                            <td>{{item.tag}}</td>
+                            <td>{{item.status}}</td>
```

Comment:
Como sugerencia, podrías utilizar la directiva `ngIf` para colocar a manera de texto el `status` de la categoría, asignando 1 a _"Activo"_ y 0 a _"Inactivo"_ como sigue:
```suggestion
                        <td> @if (category.status) {
                            Activo
                            } @else {
                            Inactivo
                            }
                        </td>
```